openmeetings-core/src/main/java/org/apache/openmeetings/core/remote/StreamProcessor.java

```diff
@@ -442,9 +446,10 @@ public void stopRecording(Client c) {
 		});
 	}
 
-	void startConvertion(Recording rec) {
+	public boolean startConvertion(Recording rec) {
```

Comment:
Needs a return value in order to have a Mockity.Spy annotation mock + verify the mock has been called.

Reply:
does it need to be `public`?

Reply:
Right, I think package private should also work. Let me check